notebooks/automated-checkout.ipynb

```diff
@@ -0,0 +1,950 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "a2e7d62b-5779-4211-822c-457c77321f8b",
+   "metadata": {},
+   "source": [
+    "# Automated checkout using yolov8, text detection and recognition models with OpenVINO™\n",
+    "\n",
+    "This Jupyter notebook demonstrates an automated checkout system that integrates object detection and text detection/recognition AI models. Utilizing YOLOv8 for object detection, the notebook efficiently identifies various items within a checkout frame, leveraging its robust and precise detection capabilities. To complement this, the notebook incorporates text detection and recognition within designated regions of interest. This combination allows for the extraction and interpretation of textual information—such as product labels and prices—from the detected items, facilitating a seamless and automated checkout process. The notebook is designed with detailed code explanations and visual outputs, making it accessible for users to understand and adapt the technology for various retail environments.\n",
+    "\n",
+    "The tutorial consists of the following steps:\n",
+    "- Download the necessary models\n",
+    "- Convert the models to OpenVINO IR.\n",
+    "- Load models to OpenVINO engine \n",
+    "- Live demo"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "d7a12678-b12f-48d1-9735-398855733e46",
+   "metadata": {},
+   "source": [
+    "## Get PyTorch model\n",
```

Comment:
so this is pytorch based and not tensorflow ?

Reply:
yes, yolov8 is pytorch based